### DISCUSSION TITLE
How to exclude models from prisma db push?

### DISCUSSION THREAD

**Discussion by bnssoftware** [prisma/prisma#14161] [Q&A]
I have a need to reference SQL Views with Prisma and followed this [article ](https://www.prisma.io/docs/guides/database/advanced-database-tasks/sql-views-postgres)to first create the view in the database and then manually create the corresponding model in prisma.schema.

But now the problem is how to exlude that manually created model from prisma db push?  I obviously don't want Prisma to create a table for this the next time I update the database schema.

**Comment by ludralph** [comment]
Hi @bnssoftware 👋,

At the moment, there is no way to exclude a model when prototyping with `db push`. 

However, If your goal is to exclude certain models from the [Prisma Client generation](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client), you can add the [@@ignore attribute](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#ignore-1) to the model definition in your Prisma schema. You can send a feature request using this [template](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and our team will look into it.

**Reply by janpio** [reply]
We don't have a good solution for the view problem yet @bnssoftware, especially when you are using `db push`. You would need to remove the "fake model" for the view before you run the command each time. We'll hopefully be able to work on this soon.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
